docs/DeveloperGuide.adoc

```diff
@@ -718,30 +718,175 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="99%",cols="2%,10%,<13%,<27%,<38%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|No. |Priority |As a ... |I want to ... |So that I can...
+|1 |`* * *` |new user |see user usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|2 |`* * *` |user |add a new person | store the information about that person
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|3 |`* * *` |user |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|4 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|5 |`* * *` |user |find my friends through their initials |find my friend’s details easily
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|6 |`* * *` |user |undo and redo my commands |undo and redo my recent errors
 
-{More to be added}
+|7 |`* * *` |user |edit my addressBook |make changes to the addressBook
+
+|8 |`* * *` |user |view my contact list in alphabetical order |scroll through the list easily
+
+|9 |`* * *` |user |use alias for commands |use the commands conveniently
+
+|10 |`* * *` |user |find person using tags |find persons based on the different categories
+
+|11 |`* * *` |user |save my person's DOB |track persons' birthdays
```

Comment:
Change `my person` to `contact` or `person`. `My person` sounds a tad too harsh and possessive.